awx/main/apps.py

```diff
@@ -63,3 +65,5 @@ def ready(self):
 
         self.load_credential_types_feature()
         self.load_named_url_feature()
+
+        discover_available_cloud_provider_plugin_names()
```

Comment:
xref https://github.com/ansible/awx/pull/15537#discussion_r1775898925

Reply:
I am not sure what xref means in this context, but I believe this is resolved. Let me know if not!

Reply:
Oh, just “external reference”. I linked it because the line got changed, and the prior thread got harder to find.

Reply:
Thank you so much for explaining! I will go ahead and resolve this now.